tools/evm/JsonTypes/TransactionInfo.cs

```diff
@@ -0,0 +1,79 @@
+using Ethereum.Test.Base;
+using Nethermind.Core;
+using Nethermind.Core.Eip2930;
+using Nethermind.Core.Crypto;
+using Nethermind.Core.Test.Builders;
+using Nethermind.Crypto;
+using Nethermind.Int256;
+
+namespace Evm.JsonTypes
+{
+    public class TransactionInfo
```

Comment:
can `TransactionForRpc` be reused?

Reply:
I tried to use TransactionForRpc, but did some modifications, added some fields, but some tests are failing